edge/pkg/devicetwin/dtclient/deviceattr_db.go

```diff
@@ -20,43 +21,43 @@ type DeviceAttr struct {
 //SaveDeviceAttr save device attributes
 func SaveDeviceAttr(doc *DeviceAttr) error {
 	num, err := dbm.DBAccess.Insert(doc)
-	log.LOGGER.Debugf("Insert affected Num: %d, %s", num, err)
+	klog.Infof("Insert affected Num: %d, %s", num, err)
 	return err
 }
 
 //DeleteDeviceAttrByDeviceID delete device attr
 func DeleteDeviceAttrByDeviceID(deviceID string) error {
 	num, err := dbm.DBAccess.QueryTable(DeviceAttrTableName).Filter("deviceid", deviceID).Delete()
 	if err != nil {
-		log.LOGGER.Errorf("Something wrong when deleting data: %v", err)
+		klog.Errorf("Something wrong when deleting data: %v", err)
 		return err
 	}
-	log.LOGGER.Debugf("Delete affected Num: %d, %s", num)
+	klog.Infof("Delete affected Num: %d, %s", num)
 	return nil
 }
 
 //DeleteDeviceAttr delete device attr
 func DeleteDeviceAttr(deviceID string, name string) error {
 	num, err := dbm.DBAccess.QueryTable(DeviceAttrTableName).Filter("deviceid", deviceID).Filter("name", name).Delete()
 	if err != nil {
-		log.LOGGER.Errorf("Something wrong when deleting data: %v", err)
+		klog.Errorf("Something wrong when deleting data: %v", err)
 		return err
 	}
-	log.LOGGER.Debugf("Delete affected Num: %d, %s", num)
+	klog.Infof("Delete affected Num: %d, %s", num)
```

Comment:
can we log these with V3 or V4 level , since these are low level logs which could be useful for debugging ?

Reply:
ok, thank you